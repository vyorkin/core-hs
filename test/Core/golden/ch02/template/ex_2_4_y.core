main = I I I I 8
