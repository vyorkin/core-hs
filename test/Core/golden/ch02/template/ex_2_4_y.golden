1 : NSupercomb main
14 : NAp 12 13
12 : NAp 11 2
14 : NAp 12 13
11 : NAp 10 2
12 : NAp 11 2
14 : NAp 12 13
10 : NAp 2 2
11 : NAp 10 2
12 : NAp 11 2
14 : NAp 12 13
2 : NSupercomb I
10 : NAp 2 2
11 : NAp 10 2
12 : NAp 11 2
14 : NAp 12 13
2 : NSupercomb I
11 : NAp 10 2
12 : NAp 11 2
14 : NAp 12 13
2 : NSupercomb I
12 : NAp 11 2
14 : NAp 12 13
2 : NSupercomb I
14 : NAp 12 13
13 : NNum 8
Stats:
  Total number of steps: 9